### PR TITLE
Feature/organizer

### PR DESCRIPTION
**Pull request by Afaed**

### CODE REVIEW

**Review by Naomilounsbury — CHANGES_REQUESTED** [review]
um you are changing 10 files here. This is too much for adding a line or two to the html

File: utils/auth.js [hunk 1]
```diff
@@ -7,17 +9,17 @@ const withAuth = (req, res, next) => {
...
-    (!req.session.user_id, req.session.activity_id === activity.organizer_id)
+    (!req.session.user_id === req.activity.organizer_id)
   ) {
-    res.redirect("/homepage");
-  } else {
-    next();
+   next()
   }
-}; */
+  res.redirect("/homepage");
```
Comment (jdpasternak) [inline utils/auth.js:19]: This should remain in the conditional statement

File: views/activity.handlebars [hunk 2]
```diff
@@ -56,7 +56,7 @@
       </div>
     </div>
   </div>
-
+<p>Organizer: {{activity.organizer_id}}</p>
```
Comment (jdpasternak) [inline views/activity.handlebars:59]: This should be `activity.user.username`

**Review by jdpasternak — CHANGES_REQUESTED** [review]
Please review comments and make appropriate changes

**Pull request closed without merging**